libsolidity/lsp/LanguageServer.cpp

```diff
@@ -20,6 +20,7 @@
 #include <libsolidity/ast/ASTVisitor.h>
 #include <libsolidity/interface/ReadFile.h>
 #include <libsolidity/interface/StandardCompiler.h>
+#include <libsolidity/interface/ImportRemapper.h>
```

Comment:
includes are grouped by category and per category alphabetically sorted. I think this line should go 2 lines up, above ReadFile, if I'm not mistaken.